website/config/custom.js

```diff
@@ -109,7 +109,7 @@ module.exports.custom = {
     'handbook/product': ['noahtalerman', 'zhumo'],
     'handbook/security': 'guillaumeross',
     'handbook/customers': ['alexmitchelliii','zayhanlon'],
-    'handbook/marketing': ['mike-j-thomas','chris-mcgillicuddy', 'timmy-k'],
+    'handbook/marketing': ['mike-j-thomas','chris-mcgillicuddy', '@mikermcneil'],
```

Comment:
I'm already on `handbook` (see line 105), so I can be left off here

Reply:
Ok sounds good. Eric and I couldn't remember what we did last time in this type of situation, so we thought this was the safest answer.